core/src/main/resources/reference.conf

```diff
@@ -219,8 +235,8 @@ system {
   instrumentation-rate = 5 seconds
 
   job-rate-control {
-    jobs = 50
-    per = 1 second
+    jobs = 20
+    per = 10 seconds
```

Comment:
Should these config defaults apply to all Cromwells or should they be overrides specifically for Terra? I'm worried that they could negatively impact someone's standalone Cromwell.

Reply:
IMHO this is probably a saner default for a "real" backend (the Dummy backend overrides this to 150 jobs / sec). Per standup today there should be a separate ticket / PR for updating Terra which is already overriding these settings.